build.sbt

```diff
@@ -282,7 +282,7 @@ lazy val interfaces = project
       V.lsp4j
     ),
     crossVersion := CrossVersion.disabled,
-    javacOptions in (Compile / doc) ++= List(
+    ((Compile / doc) / javacOptions) ++= List(
```

Comment:
just out of curiosity but are the double parens here needed? Or is it the result of the automatic rewrite?

Reply:
So I actually had the same question about pretty much all of the `()` and I'm pretty sure it's done for consistency across the board. Even the simple ones like:

```scala
(Compile / unmanagedSourceDirectories) ++= thing
```

Could just be:

```scala
Compile / unmanagedSourceDirectories ++ thing
```

So it seems that everything is just wrapped to avoid confusion. That's my assumption anyway. It was done this way with the rewrite.

Reply:
So I was wrong with my assumption. Do we have a preference on this? Do we want things wrapped, or would we prefer them not? 1, 2, 3, what do you prefer?

1.
```scala
((Compile / doc) / scalacOptions)
```

2.
```scala
(Compile / doc) / scalacOptions
```

3.
```scala
Compile / doc / scalacOptions
```

Reply:
I'm inclined to go style 3 so I'll push that up and wait to see if anyone complains. Thanks for pointing this out 👍🏼